catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -130,8 +151,22 @@ public CswRecordCollection readFrom(Class<CswRecordCollection> type, Type generi
         } finally {
             IOUtils.closeQuietly(inStream);
         }
-
         return cswRecords;
     }
 
+    /* Check HTTP headers for filename */
+    private String handleContentDispositionHeader(String contentDispositionHeader) {
```

Comment:
I think my personal preference here would be to pass in the `httpHeaders` instead of grabbing the contentDispostion header and pulling element 0 in the call to this method. It's basically a tossup but it prevents future developers who touch this class from incorrectly misusing this private method but passing in a random String here.